File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1555.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    int n;
    cin >> n;
    
    while(n--){
        int x, y;
        cin >> x >> y;
        
        int rafael = 9*x*x+y*y;
        int beto = 2*x*x+25*y*y;
        int carlos = -100*x+y*y*y;
        
        if(rafael > beto){
            if(rafael > carlos) cout << "Rafael";
            else cout << "Carlos";
        } else{
            if(beto > carlos) cout << "Beto";
            else cout << "Carlos";
        }
        
        
        cout << " ganhou" << endl;
        
    }
    
    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1555.py
n = int(input())
    
for i in range(n):
    k = [int(i) for i in input().split()]
        
    x = k[0]
    y = k[1]
        
    rafael = 9*x*x+y*y;
    beto = 2*x*x+25*y*y;
    carlos = -100*x+y*y*y;
        
    if(rafael > beto):
        if(rafael > carlos): print("Rafael", end = '')
        else: print("Carlos", end = '')
    else:
        if(beto > carlos): print("Beto", end = '')
        else: print("Carlos", end = '')
    
    print(" ganhou");
